geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingFunction.java

```diff
@@ -75,9 +77,19 @@ public CliFunctionResult executeFunction(FunctionContext<Object[]> context)
    * and the given async-event-queue as one of its queues.
    */
   private void alterRegion(Region<?, ?> region, String queueName, boolean synchronous) {
-    region.getAttributesMutator().setCacheLoader(new JdbcLoader());
+    boolean isProxy = false;
+    if (region instanceof PartitionedRegion) {
+      isProxy = ((PartitionedRegion) region).getLocalMaxMemory() == 0;
+    } else {
+      isProxy = ((LocalRegion) region).isProxy();
```

Comment:
this is an :"empty" region which is a proxy to non-pr regions in the cluster.
I didn't see you test this. I only saw the test do PR testing

Reply:
I added test for replicated region now.